src/components/bookings.tsx

```diff
@@ -59,6 +61,8 @@ const Trans: React.SFC<TransactionProps> = ({
           {transaction.pedelecName.replace(/_[nN]/g, '')}
           {', '}
           {moment(endDate, undefined, language).from(startDate, true)}
+          {transaction.credited &&
+            `, ${BUCHUNGEN.HISTORIE.STATION_PANEL.REFUNDED}`}
```

Comment:
Maybe all of this could now be changed to something like `{[value1, value2, value3].join(", ")}` in a JSX node 🤔

Reply:
Ideally, we would add a new line like the Velo app does, but that is not easily done with the `react-virtualized` dep. Additionally, its developer suggests https://github.com/bvaughn/react-window now.

Reply:
I think it's cleaner the way it is now, as conditionaly join is just as verbose. But whatever you prefer.

Reply:
`react-window` crucially does not support `WindowScroller` anymore which is why I want to stick with this library for now. There would be a way to have the current behavior with `react-window`, but we'd need to adapt the layouting a little (such that the content completely fills the window and scrolling inside the content feels like scrolling the window.